pom.xml

```diff
@@ -1432,24 +1431,6 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-remote-resources-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>process-remote-resources</id>
-            <goals>
-              <goal>process</goal>
-            </goals>
-            <configuration>
-              <resourceBundles>
-                <resourceBundle>org.apache:apache-jar-resource-bundle:1.0</resourceBundle>
-              </resourceBundles>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-
```

Comment:
I think it's good personally but is it ok to remove it? In my understanding, it looks like a license issue. If it's ok, I agree with you totally! :-)

Reply:
I think it is fine to remove this extra step in Zeppelin's pom.xml, since the execution in each Zeppelin module is triggered due to Zeppelin's parent project `maven-apache-parent`.
Take a look into https://github.com/apache/maven-apache-parent/blob/apache-26/pom.xml#L316-L333

Due to the global execution, the license is still stored in the JARs and nothing changes.

Reply:
You will see the duplicate execution of this plugin in each build.
e.g. https://github.com/apache/zeppelin/actions/runs/3523479026/jobs/5907688312#step:7:44
```
[INFO] --- maven-remote-resources-plugin:1.7.0:process (process-resource-bundles) @ zengine-plugins-parent ---
[INFO] Preparing remote bundle org.apache:apache-jar-resource-bundle:1.4
[INFO] Copying 3 resources from 1 bundle.
[INFO] 
[INFO] --- maven-remote-resources-plugin:1.7.0:process (process-remote-resources) @ zengine-plugins-parent ---
[INFO] Preparing remote bundle org.apache:apache-jar-resource-bundle:1.0
[INFO] Copying 2 resources from 1 bundle.
```

Reply:
Thank you for the explanation in detail!